src/changelog/2025/03/topic-deletion.md

```diff
@@ -0,0 +1,14 @@
+---
+title: MQTT Topic Management
+description: "MQTT Topics can now be removed from the Topic Hierarchy"
+date: 2025-03-12 12:00:00.0  
+authors: ["stephen-mclaughlin"]
+tags:
+  - changelog
+---
+
+We are always looking for ways to improve FlowFuse and today, we have made a small but notable addition: the ability to delete old topics from your hierarchy.
```

Comment:
```suggestion
We are always looking for ways to improve FlowFuse and today, we have made a small but notable addition: the ability to delete old topics from your hierarchy.

This helps you keep things tidy and not clutter your hierarchy with old topics. Note that if the topic is published to again and detected by the platform, it will get added back.
```